views/personalized-lineup.handlebars

```diff
@@ -1,10 +1,11 @@
 <h2 style="text-align: center;">Your personal {{festivalDisplayName}} lineup</h2>
 <div class="pure-g" style="justify-content: center; align-items: center;">
     <div class="pure-u-11-12 pure-u-lg-7-12 pure-u-md-5-12" style="text-align: center;">
```

Comment:
I put these backwards - the smallest fraction should be the largest screen size. Will you switch `pure-u-lg-7-12` to `pure-u-md-7-12` and `pure-u-md-5-12` to `pure-u-lg-5-12` please. That makes the wide text field below a little better

Reply:
Noted for future & done